start_admin_sshd.sh

```diff
@@ -65,33 +71,33 @@ if trusted_user_ca_keys=$(get_user_data_keys "trusted_user_ca_keys"); then
   ((++available_auth_methods))
```

Comment:
Oh, now that the counter is an int, these can become: `available_auth_methods+=1`. This isn't necessary, but it is a related cleanup item!

Reply:
I think we tried this back when we first added it, but it didn't cooperate with `set -e`. At least I remember that being the case with `((available_auth_methods++))`. If you're not strongly opposed I'd rather keep as is.

Reply:
We tried `((inside++))` which does exit non zero on 0->1 transitions - this behaves coherently:

```bash
trap 'echo exit=$? foo=$foo' DEBUG

declare -i foo=0
false # the only non-zero exit below
foo+=1
foo+=1
echo foo=$foo # should be 2!

#stdout: exit=0 foo=
#stdout: exit=0 foo=0
#stdout: exit=1 foo=0
#stdout: exit=0 foo=1
#stdout: exit=0 foo=2
#stdout: foo=2
```

Reply:
Also: fine to leave as is 👍🏽

Reply:
Ah nifty! I'm still a fan of the `++`, but this gave me a stronger appreciation for the `declare -i`. I'll be sure to keep that up in the future.